src/test/java/io/cdap/plugin/gcp/bigquery/util/SimpleFailureCollector.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.plugin.gcp.bigquery.util;
+
+import io.cdap.cdap.etl.api.FailureCollector;
+import io.cdap.cdap.etl.api.validation.ValidationException;
+import io.cdap.cdap.etl.api.validation.ValidationFailure;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nullable;
+
+public class SimpleFailureCollector implements FailureCollector {
```

Comment:
I don't think this is needed, can you remove it?

Reply:
good catch. Done :)